tools/pythonpkg/src/jupyter/jupyter_progress_bar_display.cpp

```diff
@@ -20,11 +25,20 @@ JupyterProgressBar::JupyterProgressBar() {
 	display_attr(progress_bar);
 }
 
-void JupyterProgressBar::Update(double progress) {
+JupyterProgressBarDisplay::JupyterProgressBarDisplay() : ProgressBarDisplay() {
+	// Empty, we need the GIL to initialize, which we don't have here
```

Comment:
We could also grab the GIL here, but we really don't need the initialization to happen here, I figured reducing GIL grabbing would be preferable